bundles/org.openhab.binding.vizio/src/main/resources/OH-INF/thing/vizio.xml

```diff
@@ -0,0 +1,117 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="vizio"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Vizio TV Thing -->
+	<thing-type id="vizio_tv">
+		<label>Vizio TV</label>
+		<description>
+			A Vizio SmartCast TV
+		</description>
+
+		<channels>
+			<channel id="power" typeId="system.power"/>
+			<channel id="volume" typeId="system.volume"/>
+			<channel id="mute" typeId="system.mute"/>
+			<channel id="source" typeId="source"/>
+			<channel id="activeApp" typeId="activeApp"/>
+			<channel id="control" typeId="control"/>
+			<channel id="button" typeId="buttonTv"/>
+		</channels>
+
+		<properties>
+			<property name="Model Number">unknown</property>
```

Comment:
Camel case here?
```suggestion
			<property name="modelNumber">unknown</property>
```

Or perhaps consider the name "modelId" which is framework standard: `Thing.PROPERTY_MODEL_ID`

Reply:
I will change it, but I have seen it done both ways... And this way looks better when displayed in the UI:
![image](https://user-images.githubusercontent.com/26532542/205757102-9030b757-8252-432d-aea4-b5a3934ad608.png)

Reply:
Properties have quite consistently followed the same naming convention (camelCase) in all the bindings I have seen, and also in the naming of the standard properties:
https://github.com/openhab/openhab-core/blob/cabb3f73150d13f44d647f46af8362e396684b92/bundles/org.openhab.core.thing/src/main/java/org/openhab/core/thing/Thing.java#L43-L58

So I think it would be best to rename it for consistency across the system.